### PR TITLE
Change a few fee/weight parameters

### PR DESCRIPTION
**Pull request by kianenigma**
Closes https://github.com/paritytech/polkadot/issues/2464

### Fee 

You can run the `test_fees` to get a sense of why this change is not a big deal as long as the chain is not congested. Nonetheless, this will make the chain react faster to an increase of weight, if it happens. To that end, I am also considering proposing increasing the `variability_factor` of `TargetedFeeAdjustment`. But I thik for that I should first propose it to the council for review. 

These logs are for the weight-fee of one `transfer_keep_alive` call:

```
// Polkadot 
weight = "189,060,000" // multiplier = FixedU128(0.001000000000000000) // full transfer fee = "46,015,124"
weight = "189,060,000" // multiplier = FixedU128(1.000000000000000000) // full transfer fee = "61,124,800"
weight = "189,060,000" // multiplier = FixedU128(0.001000000000000000) // full transfer fee = "46,015,124"
weight = "189,060,000" // multiplier = FixedU128(0.000001000000000000) // full transfer fee = "46,000,015"
weight = "189,060,000" // multiplier = FixedU128(0.000000001000000000) // full transfer fee = "46,000,000"

// Kusama
weight = "188,971,000" // multiplier = FixedU128(0.001000000000000000) // full transfer fee = "766,918,387"
weight = "188,971,000" // multiplier = FixedU128(1.000000000000000000) // full transfer fee = "1,018,627,759"
weight = "188,971,000" // multiplier = FixedU128(0.001000000000000000) // full transfer fee = "766,918,387"
weight = "188,971,000" // multiplier = FixedU128(0.000001000000000000) // full transfer fee = "766,666,677"
weight = "188,971,000" // multiplier = FixedU128(0.000000001000000000) // full transfer fee = "766,666,426"
```

### Weight

A new analysis of weights show that the average weight weight is still 6B and 8B for polkadot and kusama respectively: 

Screenshot 2021-02-19 at 11 05 33

Which means the average `on_initialize` is 0.4% and 0.3% respectively, so we set it now to 1%. Note that this is again mostly to make the nominator community of polkadot temporarily bigger, while PRs to fix this fundamentally arrive. Afterwards, we can go back to numbers like 5% again.

### CODE REVIEW

File: runtime/common/src/lib.rs [hunk 1]
```diff
@@ -77,7 +77,7 @@ parameter_types! {
 	/// Minimum amount of the multiplier. This value cannot be too low. A test case should ensure
 	/// that combined with `AdjustmentVariable`, we can recover from the minimum.
 	/// See `multiplier_can_grow_from_zero`.
-	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000_000u128);
+	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000u128);
```
Comment (shawntabrizi) [inline runtime/common/src/lib.rs:80]: Even if this is the correct value, I might prefer we reduce it by 1,000 over 2 upgrades than by 1,000,000 at once :)

Comment (kianenigma) [reply runtime/common/src/lib.rs:80]: cowboy prevention system activated 🤠 

fair enough.

Comment (kianenigma) [inline runtime/common/src/lib.rs:80]:
```suggestion
	pub MinimumMultiplier: Multiplier = Multiplier::saturating_from_rational(1, 1_000_000u128);
```

**Review by shawntabrizi — APPROVED** [review]
cool

Maybe make another issue to reduce by 1,000 again if needed

**Review by thiolliere — APPROVED** [review]
I don't know if it is really needed to increase the minimum multiplier.
If chain is congested the fast mechanism is tipping.
Otherwise looks good to me

**Pull request merged**